features/keychain/module/crypto/secp256k1.js

```diff
@@ -6,9 +7,11 @@ export const create = ({ getPrivateHDKey }) => {
   const curve = new EC('secp256k1')
 
   const createInstance = () => ({
-    signBuffer: async ({ seedId, keyId, data }) => {
+    signBuffer: async ({ seedId, keyId, data, ecOptions, enc = 'der' }) => {
+      assert(['der', 'raw'].includes(enc), 'signBuffer: invalid enc')
       const { privateKey } = getPrivateHDKey({ seedId, keyId })
-      return Buffer.from(curve.sign(data, privateKey).toDER())
+      const signature = curve.sign(data, privateKey, ecOptions)
+      return enc === 'der' ? Buffer.from(signature.toDER()) : { ...signature }
```

Comment:
It's worth noting that the recovery id (`recid` or `v` in eth ecosystem) can be derived from `r` and `s` but it [may take up to 4 tries](https://github.com/indutny/elliptic/blob/master/lib/elliptic/ec/index.js#L226-L243). A consumer of this API would be [able to canonicalize the `s` value itself](https://github.com/indutny/elliptic/blob/master/lib/elliptic/ec/index.js#L147-L149) & compute `recid` itself but these seem like good quality-of-life changes.